storops_test/unity/resource/test_filesystem.py

```diff
@@ -245,6 +245,16 @@ def test_modify_success_empty_description(self):
         fs.modify(description='')
         assert_that(fs.description, equal_to(''))
 
+    @patch_rest
+    def test_modify_success_snap_schedule(self):
+        fs = UnityFileSystem(cli=t_rest(), _id='fs_22')
+
+        snapScheduleParameters = {'snapSchedule': {'id': 'snapSch_1'}, 'isSnapSchedulePaused': False}
+
+        fs.modify(snapScheduleParameters=snapScheduleParameters)
+        fs_snapSchedule = fs.snapScheduleParameters['snapSchedule']
+        assert_that(fs_snapSchedule['id'], equal_to('snapSch_1'))
```

Comment:
Test failed with:

```
E           MockFileNotFoundError: cannot find response for url: /api/instances/storageResource/res_22/action/modifyFilesystem?compact=True, 
E           body: {u'snapScheduleParameters': {u'isSnapSchedulePaused': False, u'snapSchedule': {u'id': u'snapSch_1'}}}
storops_test/unity/rest_mock.py:99: MockFileNotFoundError
```

Need to update index.json in `storops_test/unity/rest_data/storageResource/`

Reply:
Done